src/D0_modelling_and_optimization.py

```diff
@@ -111,6 +112,15 @@ def warning_asset_type(asset, type, assetGroup):
 
     logging.info("Adding constraints to oemof model...")
     # todo include constraints
+    if "Electrolyser" in dict_model["transformers"]:
+        model_constraints.standby_consumption(
+            local_energy_system,
+            dict_model["busses"]["Electricity bus"],
+            dict_model["busses"]["H2 bus"],
+            dict_model["transformers"]["Electrolyser"],
+            dict_model["sinks"]["Electrolyser standby_sink"],
+            dict_values["energyConversion"]["electrolyser"]["efficiency"]["value"],
+        )
```

Comment:
You here have "electrolyzer" and "Electrolyzer" (capitalized) - why is this differing? Hard coded links are though for later use.